Setup.hs

```diff
@@ -1,11 +1,15 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE TupleSections #-}
 
+#ifndef MIN_VERSION_Cabal
```

Comment:
That's right. Older versions of Cabal don't know what that section is and will issue a warning and ignore it.